guides/common/modules/proc_changing-email-notification-settings-for-a-host.adoc

```diff
@@ -5,12 +5,12 @@
 You can configure {Project} to send email notifications either to an individual user or a user group.
 When set to a user group, all group members who are subscribed to the email type receive a message.
 
-To view the notification status for a host, navigate to *Hosts* > *All Hosts* and click the host you want to view.
-In the host details page, click the *Additional Information* tab, you can view the email notification status. 
-
 Receiving email notifications for a host can be useful, but also overwhelming if you are expecting to receive frequent errors, for example, because of a known issue or error you are working around.
-To change the email notification settings for a host, complete the following steps.
+To view or change the email notification settings for a host, complete the following steps.
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*, and select the host with the notification setting you want to change.
-. Select the host's checkbox, and from the *Select Action* list, select *Enable Notifications* or *Disable Notifications*, depending on what you want.
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*, locate the host that you want to view, and click *Edit* in the *Actions* column.
+. Go to the *Additional Information* tab.
+If the checkbox *Include this host within {Project} reporting* is checked, then the email notifications are enabled on that host.
```

Comment:
```suggestion
. On the *Additional Information* tab, check the checkbox *Include this host within {Project} reporting* to enable email notifications.
If enabled, the owner of the host receives email notifications for that host.
```

Unsure about the second sentence. IMHO we should also refer users to Administer > Users > Your User > email tab which determines if a users receives emails at all.

Reply:
I can add that, but the checkbox is checked by default (at least for admins) so I don't want to explicitly instruct users to check it.

Reply:
Added note.